app/_plugins/konnect_icon_tag.rb

```diff
@@ -0,0 +1,15 @@
+module Jekyll
+  class KonnectIconTag < Liquid::Tag
+
+    def initialize(tag_name, icon, tokens)
+      super
+      @icon = icon
+    end
+
+    def render(context)
+      "![#{@icon} icon](/assets/images/icons/konnect/icn-#{@icon}.svg){:.inline .konnect-icn .no-image-expand}"
```

Comment:
Looks like this is adding an extra space in the filename, so it's generating a broken image:

![Screen Shot 2022-03-30 at 12 45 05 PM](https://user-images.githubusercontent.com/54370747/160918194-1d01b8ff-a45f-4b73-ab09-e16ee950bf4a.png)

https://deploy-preview-3798--kongdocs.netlify.app/konnect/